### ISSUE TITLE
MSBuild fails when subject name in App manifest file has special character in CN: Validation error. error C00CE169:

### ISSUE THREAD

**Issue by V-M-Aravind** [dotnet/msbuild#10512]
`Command failed with error MSBuildError: 12:6>D:\abcapp\windows\x64\Release\ABCAPP\AppxManifest.xml : error APPX0501: Validation error. error C00CE169: App manifest validation error: The app manifest must be valid as per schema: Line 13, Column 57, Reason: 'C=US, ST=Texas, L=Kerrville, O=ABCD, INC., CN=ABCD, INC.' violates pattern constraint of '(CN|L|O|OU|E|C|S|STREET|T|G|I|SN|DC|SERIALNUMBER|Description|PostalCode|POBox|Phone|X21Address|dnQualifier|(OID\.(0|[1-9][0-9]*)(\.(0|[1-9][0-9]*))+))=(([^,+="<>#;])+|".*")(, ((CN|L|O|OU|E|C|S|STREET|T|G|I|SN|DC|SERIALNUMBER|Description|PostalCode|POBox|Phone|X21Address|dnQualifier|(OID\.(0|[1-9][0-9]*)(\.(0|[1-9][0-9]*))+))=(([^,+="<>#;])+|".*")))*'. The attribute 'Publisher' with value 'C=US, ST=Texas, L=Kerrville, O=ABCD, INC., CN=ABCD, INC.' failed to parse. [D:\abccd\windows\ABCAPP\ABCAPP.vcxproj]`

Subject name in my certificate is **C=US, ST=Texas, L=Kerrville, O=ABCD, INC., CN=ABCD, INC.** , where O and CN has comma in it.
How should i format this to work?

**Comment by rainersigwald** [comment]
Team Triage: We believe this is an issue with a task that ships in Visual Studio, not MSBuild itself. Can you please [open a feedback ticket instead](https://docs.microsoft.com/visualstudio/ide/feedback-options)?

After it's created, that will open an internal bug which will be routed to the right team. If you post the link here we may be able to help that process go faster.

**Comment by V-M-Aravind** [comment]
@rainersigwald , I also tried omitting the O and CN, then also I get the same error. This time it's due to ST. If I replace ST with S and omitting the O and CN, it will work. So in short ST and comma is the source of the issue.

**Comment by V-M-Aravind** [comment]
I signed my code. This is what I did for my package manifest file. So the subject name of the certificate is like this:
CN=ABCD, INC. , O=ABCD, INC., L=Kerrville, ST=Texas, C=US
In Package.manifest file I added like this :  
Publisher="CN=&quot;ABCD, INC.&quot;, O=&quot;ABCD, INC.&quot;, L=Kerrville, S=Texas, C=US"
I wrapped the attributes having comma in quotes and escaped them. I aslo changed ST to S in package.manifest file. This worked. Hence closing it.
